ui/app/AppLayouts/Profile/Sections/NotificationsContainer.qml

```diff
@@ -5,15 +5,255 @@ import "../../../../imports"
 import "../../../../shared"
 import "../../../../shared/status"
 
-Item {
-    id: notificationsContainer
-    anchors.right: parent.right
-    anchors.rightMargin: contentMargin
-    anchors.left: parent.left
-    anchors.leftMargin: contentMargin
-
-    StatusSectionHeadline {
-      text: qsTr("Notification preferences")
-      anchors.top: parent.top
+ScrollView {
+    height: parent.height
+    width: parent.width
+    contentHeight: notificationsContainer.height
+    clip: true
+
+    Item {
+        id: notificationsContainer
+        anchors.right: parent.right
+        anchors.rightMargin: contentMargin
+        anchors.left: parent.left
+        anchors.leftMargin: contentMargin
+        height: sectionHeadlineNotifications.height * 3 + column.height + Style.current.smallPadding * 3 + Style.current.padding * 2 + Style.current.bigPadding * 4 + column2.height + column3.height + column4.height + 200 + column5.height
+
+        ButtonGroup {
+            id: notificationSetting
+        }
+
+        ButtonGroup {
+            id: soundSetting
+        }
+
+        ButtonGroup {
+            id: messageSetting
+        }
+
+        StatusSectionHeadline {
+            id: sectionHeadlineNotifications
+            text: qsTr("Notification preferences")
+            anchors.top: parent.top
+        }
+
+        Column {
+            id: column
+            spacing: Style.current.padding
+            anchors.top: sectionHeadlineNotifications.bottom
+            anchors.topMargin: Style.current.smallPadding
+            width: parent.width
+
+            RowLayout {
+                width: parent.width
+                StyledText {
+                    text: qsTr("All messages")
+                    font.pixelSize: 15
+                }
+
+                StatusRadioButton {
+                    checked: true
+                    Layout.alignment: Qt.AlignRight
+                    ButtonGroup.group: notificationSetting
+                    rightPadding: 0
+                }
```

Comment:
Ideally, instead of having a StyledText and StatusRadioButton combo, we could just leverage StatusRadioButton `contentItem` here. However, after fiddling with it for about an hour or so, I coudln't find a way to have the contentItem be on the left-hand side... Doesn't seem to be possible

Reply:
Ugh, yeah QT sometimes suck for that. Customizing their components is super hard.

What you can do is create another component named `StatusRadioButtonRight` that is just what you did above, i.e. a StyledText just before a `StatusRadioButton`

Reply:
For now I'll leave it as is. If there's more places where such a UI is needed, We can turn it into a component. I'd like to avoid introducing something like `StatusRadioButtonRight`, maybe there is a way to make it work as intended. 

Will revisit later